presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -170,6 +170,7 @@
     public static final String INLINE_SQL_FUNCTIONS = "inline_sql_functions";
     public static final String REMOTE_FUNCTIONS_ENABLED = "remote_functions_enabled";
     public static final String CHECK_ACCESS_CONTROL_ON_UTILIZED_COLUMNS_ONLY = "check_access_control_on_utilized_columns_only";
+    public static final String SKIP_REDUNDANT_SORT = "remove_redundant_sort";
```

Comment:
the string should be skip_redundant_sort to match the config property and variable name.

Reply:
yes they should be the same,fixed